.ci/azure-pipelines.yml

```diff
@@ -63,11 +63,11 @@ jobs:
       matrix:
         XCode-latest:
           test_config: 'ci'
-          vmImage: 'macOS-latest'
+          vmImage: 'macOS-12'
```

Comment:
is there a macOS-13 available?

Reply:
`macos-latest` still points to `macos-11`

macOS 13 is in beta. It probably won't become available as a runner for CI builds until next June if the timeline taken for macos-12 runners is followed; there might be another closed testing period that we could apply for to get access sooner.